include/certifier_algorithms.h

```diff
@@ -69,7 +69,7 @@ extern const char *Integrity_method_aes_256_gcm;
 extern const char *Integrity_method_hmac_sha256;
 
 extern const char *Enc_public_key_algorithms[];
-extern const char *Enc_symmetric_key_algorithms[];
+extern const char *Enc_authenticated_symmetric_key_algorithms[];
```

Comment:
Renamed name of array of string constants, to explicitly call-out the 'Authenticated' part of these algorithms.